src/App.tsx

```diff
@@ -15,48 +16,114 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  reverse: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, reverse }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  // eslint-disable-next-line no-console
+  console.log(sortType, reverse);
+
+  return visibleGoods;
+}
```

Comment:
for now you have function wich just return copy of goods, why do you need second arg?

remove console.log
remove comments